apps/meteor/ee/server/api/sessions.ts

```diff
@@ -0,0 +1,167 @@
+import { Meteor } from 'meteor/meteor';
+import { check, Match } from 'meteor/check';
+
+import { hasRole } from '../../../app/authorization/server';
+import { Users, Sessions } from '../../../app/models/server/raw/index';
+import { API } from '../../../app/api/server/api';
+import { hasPermission } from '../../../app/authorization/server/functions/hasPermission';
+import { hasLicense } from '../../app/license/server/license';
+
+API.v1.addRoute(
+	'sessions/list',
+	{ authRequired: true },
+	{
+		async get() {
+			if (!this.userId) {
+				API.v1.failure('error-invalid-user');
+			}
+			if (!hasLicense('device-management')) {
+				return API.v1.unauthorized();
+			}
+			check(
+				this.queryParams,
+				Match.ObjectIncluding({
+					offset: Match.Maybe(String),
+					count: Match.Maybe(String),
+					search: Match.Maybe(String),
+				}),
+			);
+
+			try {
```

Comment:
no need for the try/catch if you're not doing something special with the error 👀